### PR TITLE
Issues Fix

### LINKED ISSUE #321: more faq button directs to old website
**To Reproduce**
1. Go to FAQ
2. Click on Feedback form btn
3. See error

**Expected behavior**
direct to new website

**Screenshots**
image

### LINKED ISSUE #322: update the licesence in more info page
**Screenshots**
image

### LINKED ISSUE #323: add separation in all the info card
image

### PR DESCRIPTION
**Pull request by Manas1820**
# Description

Please include the summary of the change as well as the issue that is fixed (if applicable). Also if any dependencies that are required for this change to work.

Fixes #322 
Fixes #323 
Fixes #321

## Type of change

Please choose the relavant type of change.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Also summarise the process for us to reproduce.

- [ ] Test A: checked through my emulator.
- [ ] Test B: Firebase testing labs
- [ ] Test C: Test through flutter test code
- [ ] Test D: Other (Please specify)

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this application
- [ ] I have checked the code working on my emulator
- [ ] I have commented my code for better understanding
- [ ] These changes generate no new warnings
- [ ] These changes do not effect the existing functionalities,

### CODE REVIEW

**Review by hayat-tamboli — CHANGES_REQUESTED** [review]
Bro change the license content too

**Comment by Manas1820** [comment]
> Bro change the license content too

done

File: lib/view/profile/drawer/drawer.dart [hunk 1]
```diff
@@ -206,4 +206,4 @@ _launchURL(String gurl) async {
   }
 }
 
-var websiteURL = "https://runbhumi-website.vercel.app/";
+var websiteURL = "https://runbhumi-website.vercel.app/feedback";
```
Comment (hayat-tamboli) [inline lib/view/profile/drawer/drawer.dart:209]: bro revert this back

**Review by hayat-tamboli — CHANGES_REQUESTED** [review]
please revert the websiteURL to "https://runbhumi-website.vercel.app/" in line 209 of lib/view/profile/drawer/drawer.dart

**Comment by Manas1820** [comment]
> please revert the websiteURL to "https://runbhumi-website.vercel.app/" in line 209 of lib/view/profile/drawer/drawer.dart

ok

**Review by hayat-tamboli — APPROVED** [review]

**Pull request merged**